packages/zendesk-adapter/test/change_validators/business_hours_schedule_holiday.test.ts

```diff
@@ -21,19 +22,21 @@ describe('businessHoursScheduleHolidayChangeValidator', () => {
     })
 
   const dateYearsAgo = (years: number): string => {
-    const date = new Date()
     date.setFullYear(date.getFullYear() - years)
-    return date.toISOString()
+    const ret = date.toISOString()
+    date.setFullYear(date.getFullYear() + years)
```

Comment:
```suggestion
    date.setFullYear(date.getFullYear()  - years)
```
as the name of the function is dateYearsAgo